src/Storage/Commands.Storage/Blob/Cmdlet/GetAzureStorageBlobContent.cs

```diff
@@ -26,7 +26,7 @@
 
 namespace Microsoft.WindowsAzure.Commands.Storage.Blob.Cmdlet
 {
-    [Cmdlet(VerbsCommon.Get, StorageNouns.BlobContent, ConfirmImpact = ConfirmImpact.High, DefaultParameterSetName = ManualParameterSet),
+    [Cmdlet(VerbsCommon.Get, StorageNouns.BlobContent, SupportsShouldProcess = true, DefaultParameterSetName = ManualParameterSet),
```

Comment:
The default is ```ConfirmImpact.Medium```, which means no confirmation when default powershell settings are used.   In the default case, the user will only be prompted (using ```ShouldContinue```) if the download would overwrite a local file.